server/index.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable no-console */
+const express = require('express');
+const db = require('../database/index');
+
+const PORT = 3000 || process.env.PORT;
+
+const app = express();
+
+app.use(express.json());
+
+app.get('/api/photos', (req, res) => {
+  db.query('SELECT * FROM photos', (err, data) => {
+    if (err) {
+      console.log(err);
+      res.sendStatus(500);
+    } else {
```

Comment:
this makes sense since you have to pull a random amount of images.

Reply:
I made one to get all the images and one where I am looking for a certain 'house/Listing_id'